src/internal/utils/relations.ts

```diff
@@ -0,0 +1,41 @@
+import { QueryParam } from "../executer.interfaces";
+
+export enum RelationLinkType {
+  ATTACH = "attach",
+}
+
+/**
+ * Adds attach operation params to a query params array.
+ *
+ * @param   queryParams The query params to be appended
+ * @param   resourceName The name of the resource
+ * @param   relationLinkType The type of t
+ * @returns QueryParam[]
+ */
+export function linkRelation(
```

Comment:
I would name it `runAction` or something similar because there are could be a lot of different actions, not about relations only

Reply:
I won't need this method anymore 👍